### PR TITLE
Make more use of the generated doc

### PR DESCRIPTION
**Pull request by gsmet**
@geoand could you have a look to this one? I would like to push it into 0.26.0.

### CODE REVIEW

File: core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java [hunk 1]
```diff
@@ -38,17 +28,44 @@
     public Level minLevel;
 
     /**
-     * Console logging config
+     * Logging categories.
+     * <p>
+     * Logging is done on a per-category basis. Each category can be independently configured.
+     * A configuration which applies to a category will also apply to all sub-categories of that category,
```
Comment (geoand) [inline core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java]: I don't really like `A configuration` and would personally replace it with `Configuration`, but that's too minor :P

Comment (gsmet) [reply core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java]: Yeah, I just took it from the guide. For now, it's not even displayed.

File: docs/src/main/java/io/quarkus/docs/generation/AllConfigGenerator.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ public static void main(String[] args)
         File jsonFile = new File("devtools/core-extensions-json/target/extensions.json");
         if (!jsonFile.exists()) {
             System.err.println("WARNING: could not generate all-config file because extensions list is missing: " + jsonFile);
-            System.exit(0);
+            return;
```
Comment (geoand) [inline docs/src/main/java/io/quarkus/docs/generation/AllConfigGenerator.java]: :+1:

**Review by geoand — APPROVED** [review]

File: core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java (hunk 1 above)
Comment (geoand) [reply core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java]: Cool

**Comment by gsmet** [comment]
Rebased and force pushed.

**Pull request merged**